app/Grouptopics/Repositories/Account/AccountEloquent.php

```diff
@@ -1,19 +1,22 @@
 <?php
-namespace Grouptopics\Libraries;
+namespace Grouptopics\Repositories\Account;
 
-use \Input as Input;
-use \Auth as Auth;
-use \User as User;
-use \Validator as Validator;
-use \Hash as Hash;
-use \Redirect as Redirect;
+use Grouptopics\Repositories\EloquentRepository;
 
-class Account
+
+class AccountEloquent extends EloquentRepository implements AccountInterface
```

Comment:
Should this be `AccountRepository` instead? Eloquent is the name of the library, repository is what this is